app/models/common_exposure_cohort.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# CommonExposureCohort: a common exposure cohort
+class CommonExposureCohort < ApplicationRecord
+  belongs_to :patient, touch: true
+
+  VALID_COHORT_TYPES = [nil, '', 'Adult Congregate Living Facility', 'Child Care Facility', 'Community Event or Mass Gathering', 'Correctional Facility',
+                        'Group Home', 'Healthcare Facility', 'Place of Worship', 'School or University', 'Shelter', 'Substance Abuse Treatment Center',
+                        'Workplace', 'Other'].freeze
+
+  validates :cohort_type, inclusion: {
+    in: VALID_COHORT_TYPES,
+    message: "is not an acceptable value, acceptable values are: '#{VALID_COHORT_TYPES.reject(&:blank?).join("', '")}'"
+  }
```

Comment:
This validation doesn't show up in the API because there is no label for this field in the `ValidationHelper`.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1125/commits/11f9765d3d0d6b687f0a3e8f6d62ab6192eadf4b#diff-4f61e388997f8d92e41f66947a47b461707b615cf9fac69f1080b8254630d2ceR219